apps/prairielearn/src/tests/sync/assessmentsSync.test.ts

```diff
@@ -98,6 +99,10 @@ async function getSyncedAssessmentData(tid: string) {
       group_roles: z.array(GroupRoleSchema),
     }),
   );
+  if (result.assessment.sync_errors) {
+    logger.error(result.assessment.sync_errors);
+  }
```

Comment:
This seems like it's doing the opposite of what we're aiming to do in #12644. Was this committed intentionally?

Reply:
Yes, I'm hoping that with #12644 I can then turn this off again.

Reply:
For tests that are failing unexpectedly, this logs the needed information for why they failed. I could also place this log within the syncing code itself. With #12644, I could then turn off the logs for any test where it logs and that was intentional, which is currently one scenario.

Reply:
Changed in [9441366](https://github.com/PrairieLearn/PrairieLearn/pull/12675/commits/94413666c4f5a93111eb9b9b77e152c122a27faf). We discussed alternative solutions for better test failures in DMs.